Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,43 +12,46 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    <div class="col-md-3">
-        <div>
-            <h4>
-                {% if admin.hasRoute('list') %}
-                    <a href="{{ admin.generateUrl('list') }}">{{ admin.label|trans({}, admin.translationdomain) }}</a>
-                {% else %}
+    <div class="col-lg-4 col-md-6">
+        <div class="box">
+            <div class="box-header with-border">
+                <h3 class="box-title">
+                    {% set icon = settings.icon|default('') %}
+                    {{ icon|raw }}
                     {{ admin.label|trans({}, admin.translationdomain) }}
-                {% endif %}
-
+                </h3>
                 {% if pager and pager.getNbResults() > 0 %}
-                    <span class="label label-info">{{ pager.getNbResults() }}</span>
+                    <span class="badge pull-right">{{ pager.getNbResults() }}</span>
                 {% endif %}
-            </h4>
-        </div>
-
-        <ul>
-            {% if pager %}
-                {% for result in pager.getResults() %}
-                    {% if admin.hasRoute('edit') %}
-                        <li><a href="{{ admin.generateObjectUrl('edit', result) }}">{{ admin.toString(result) }}</a></li>
-                    {% else %}
-                        <li><i>{{ admin.toString(result) }}</i></li>
-                    {% endif %}
-                {% else %}
-                    {% if admin.hasRoute('create') %}
-                        <li><i>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }} ~ <a href="{{ admin.generateUrl('create') }}">{{ 'add_new_entry'|trans({}, 'SonataAdminBundle') }}</a></i></li>
-                    {% else %}
-                        <li><i>{{ 'no_results_found'|trans({}, 'SonataAdminBundle') }}</i></li>
-                    {% endif %}
-                {% endfor %}
+            </div>
+            {% if pager and pager.getResults()|length %}
+                <div class="box-body no-padding">
+                    <ul class="nav nav-stacked">
+                        {% for result in pager.getResults() %}
+                            {% if admin.hasRoute('edit') %}
+                                <li><a href="{{ admin.generateObjectUrl('edit', result) }}">{{ admin.toString(result) }}</a></li>
+                            {% else %}
+                                <li><a>{{ admin.toString(result) }}</a></li>
+                            {% endif %}
+                        {% endfor %}
+                        {% if pager.getNbResults() > pager.getResults()|length %}
+                            {% if admin.hasRoute('list') %}
+                                <li><a href="{{ admin.generateUrl('list') }}"><i class="fa fa-ellipsis-h"></i></a></li>
```

Comment:
Same here. Don't know if just a list link is useful.

@rande What do you think?